pom.xml

```diff
@@ -31,26 +31,22 @@
 
   <modules>
     <module>api</module>
-    <module>api-1.11</module>
-    <module>api-2.0</module>
-    <module>omod-1.10</module>
-    <module>omod-2.0</module>
     <module>omod</module>
   </modules>
 
   <properties>
-    <openMRSVersion>1.10.5</openMRSVersion>
-    <openMRS2_0Version>2.0.0</openMRS2_0Version>
+    <openMRSVersion>2.4.0</openMRSVersion>
     <appframeworkVersion>2.2.1</appframeworkVersion>
     <uiframeworkVersion>3.21.0</uiframeworkVersion>
     <uicommonsVersion>2.16.0</uicommonsVersion>
-    <webservices.restVersion>2.17</webservices.restVersion>
+    <webservices.restVersion>2.33.0</webservices.restVersion>
     <emrapiVersion>1.19</emrapiVersion>
     <reportingVersion>0.9.2.1</reportingVersion>
     <calculationVersion>1.1</calculationVersion>
     <serialization.xstreamVersion>0.2.8</serialization.xstreamVersion>
     <providermanagementVersion>2.3</providermanagementVersion>
     <appuiVersion>1.3</appuiVersion>
+    <javaxVersion>4.0.1</javaxVersion>
```

Comment:
What do we use this for?

Reply:
I should use the property [here](https://github.com/openmrs/openmrs-module-attachments/pull/57/files#diff-ffa8aabf43bef372dc7e0df2dc08200ed66072665a552afe214a4487c1b2ef53R110) actually, specifically scoped as `test` for tests to pass.

Reply:
...not forgetting to cleanup the remaining unused properties, thanks for the review @dkayiwa.